Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2430,24 +2412,38 @@ def to_context(self):
 
         if self.indicators:
             for indicator in self.indicators:
-                outputs.update(indicator.to_context())
+                context_outputs = indicator.to_context()
+                for key in context_outputs.keys():
+                    value = context_outputs[key]
+
+                    if key not in outputs:
+                        outputs[key] = []
+
+                    outputs[key].append(value)
 
         if self.raw_response:
             raw_response = self.raw_response
 
         if self.outputs:
             if not self.readable_output:
+                # if markdown is not provided then create table by default
                 human_readable = tableToMarkdown('Results', self.outputs)
             else:
                 human_readable = self.readable_output
 
             if not self.raw_response:
                 raw_response = self.outputs
 
-            outputs_key = '{}(val.{} == obj.{})'.format(self.outputs_prefix, self.outputs_key_field, self.outputs_key_field)
-            outputs.update({
-                outputs_key: self.outputs
-            })
+            if self.outputs_prefix and self.outputs_key_field:
+                # if both prefix and key field provided then create DT key
+                outputs_key = '{}(val.{} == obj.{})'.format(self.outputs_prefix, self.outputs_key_field, self.outputs_key_field)
```

Comment:
would be nicer i think
```suggestion
                outputs_key = '{0}(val.{1} == obj.{1})'.format(self.outputs_prefix, self.outputs_key_field)
```